Uno/NugetPackageExplorer.Legacy/Presentation/ShellViewModel.cs

```diff
@@ -3,19 +3,33 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Windows.Input;
+
 using Microsoft.Extensions.DependencyInjection;
+using Uno.Foundation;
 using NupkgExplorer.Framework.MVVM;
 using NupkgExplorer.Framework.Navigation;
 using NupkgExplorer.Presentation.Content;
 
 namespace NupkgExplorer.Presentation
 {
-	public class ShellViewModel : ViewModelBase
-	{
-		public ViewModelBase ActiveContent { get => GetProperty<ViewModelBase>(); set => SetProperty(value); }
+    public class ShellViewModel : ViewModelBase
+    {
+        public ViewModelBase ActiveContent { get => GetProperty<ViewModelBase>(); set => SetProperty(value); }
+
+        public bool IsStandaloneWindow
+        {
+            get => GetProperty<bool>();
+            set => SetProperty(value);
+        }
+
+        public ShellViewModel()
+        {
 
-		public ShellViewModel()
-		{
-		}
-	}
+#if __WASM__
+            IsStandaloneWindow = WebAssemblyRuntime.InvokeJS(@"window.matchMedia('(display-mode: standalone)').matches") == "true";
+#else
+            IsStandaloneWindow = true;
+#endif
```

Comment:
This needs to be reevaluated somehow, as when the webapp is being installed as an app, the button does not unless the app is restarted. I wonder if we could mitigate this by determining that we should reevaluate this property when the size of the window changes very significantly (which happens when the app is installed).